src/main/java/com/google/sps/MockData.java

```diff
@@ -0,0 +1,387 @@
+                                            businessPhotoUrlList,
+                                            businessDescr,
+                                            menuUrl,
+                                            businessOrderInfo,
+                                            businessContactUrl,
+                                            linkToBusiness);
+
+    businessName = "Streets of Bangkok";
+    categoryList = Arrays.asList("Mains");
+    businessLat = 1.297275F;
+    businessLong = 103.78742F;
+    businessAddr = "73A Ayer Rajah Crescent, 139957, Singapore";
+    businessLogoUrl = "https://res-1.cloudinary.com/crunchbase-production/image/upload/c_lpad,h_120,w_120,f_auto,b_white,q_auto:eco/v1412921346/nnzir0xhukjpbpmctwy9.jpg";
+    businessPhotoUrlList = Arrays.asList("images/the-oven-bakes/the-oven-bakes-1.PNG","images/the-oven-bakes/the-oven-bakes-2.PNG","images/the-oven-bakes/the-oven-bakes-3.PNG","images/the-oven-bakes/the-oven-bakes-5.PNG");
+
+    businessDescr = "Serving you authentic dishes found along streets of Bangkok\n\n" +
+                    "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut" +
+                    "labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris" +
+                    "nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit" +
+                    "esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in" +
+                    "culpa qui officia deserunt mollit anim id est laborum.";
+
+    menuUrl = "https://www.instagram.com/streetsofbangkok.sg/";
+    businessOrderInfo = "Minimum Order(for delivery): $15\n\n" +
+                                "Islandwide Delivery: $10";
+    businessContactUrl = "https://www.facebook.com/streetsofbangkok/";
+    linkToBusiness = "https://www.instagram.com/streetsofbangkok.sg/";
+
+    Business mockBusiness10 = new Business(businessName, 
+                                            categoryList,
+                                            4.9F, 
+                                            businessLat, 
+                                            businessLong, 
+                                            businessAddr, 
+                                            businessLogoUrl, 
+                                            businessPhotoUrlList,
+                                            businessDescr,
+                                            menuUrl,
+                                            businessOrderInfo,
+                                            businessContactUrl,
+                                            linkToBusiness);
+
+    mockDataList = Arrays.asList(mockBusiness1, mockBusiness2, mockBusiness3, mockBusiness4, mockBusiness5, mockBusiness6, mockBusiness7, mockBusiness8, mockBusiness9, mockBusiness10);
```

Comment:
I think you can use a for loop here since you are differentiating by number? easier to scale next time! But I think this works too!

Reply:
Hmm, but since i'm refreshing the variables of the Business object every loop, and they are all distinct, there would still be a lot of code duplication either way, so I don't think it helps a lot to have a forloop to add each mock business to the array.

Reply:
do we need this to be scalable? it's just mock data. i don't think we'll prepare the data from the Datastore this way in the future